app/models/user.rb

```diff
@@ -1,4 +1,5 @@
 class User < ActiveRecord::Base
   has_many :bids
   validates :email, email: true, allow_blank: true
+  validates :duns_number, format: { with: /\A\d{8,13}\z/ }
```

Comment:
https://github.com/18F/samwise/tree/micropurchase-compatibility#validate-the-format-of-a-duns-number

^ could be used instead